x/interchainquery/keeper/msg_server.go

```diff
@@ -59,22 +59,22 @@ func (k msgServer) SubmitQueryResponse(goCtx context.Context, msg *types.MsgSubm
 
 			tmclientstate, ok := clientState.(*tmclienttypes.ClientState)
 			if !ok {
-				k.Logger(ctx).Error("error unmarshaling client state", "cs", clientState)
+				k.Logger(ctx).Error(fmt.Sprintf("error unmarshaling client state %v", clientState))
```

Comment:
was there as issue at some point with `%v` prints?

Reply:
It's also not clear to me whether %v's prints the key's or not. I think I've seen both cases. Do you know @riley-stride ?

Reply:
Not 100% sure, we should prob just be safe by not using %v [](edited-with-Axolo)

Reply:
%v uses the default print function for the type (e.g. %s for strings) and won't crash. the Error logger from the sdk crashed using %d (unclear why). 

to @sampocs point, %v won't print keys (just values), but I think that's fine (we can fix it in a future PR)

Reply:
That's what I thought too, but then I saw this in the logs
```
log = fmt.Sprintf("\t [IBC-TRANSFER] Deposit record updated: {%v}", record)
```
![Screen Shot 2022-08-13 at 8 59 10 PM](https://user-images.githubusercontent.com/19301583/184519388-a2ac49e5-618f-4174-a213-9cc542d3cbe7.png)

Reply:
cant see the screenshot [](edited-with-Axolo)

Reply:
check github [](edited-with-Axolo)

Reply:
:+1: [](edited-with-Axolo)

Reply:
oh, nice! so it does print keys